tracpro/templates/baseline/baselineterm_read.html

```diff
@@ -3,9 +3,70 @@
 
 {% block pre-content %}
 
+  {% if error_message %}
+    <div class="alert alert-danger form-errors">
+      {{ error_message }}
+    </div>
+  {% endif %}
+
   {% include 'baseline/baselineterm_header.html' %}
 
+  <div id="container" style="min-width: 310px; height: 400px; max-width: 800px; margin: 0 auto"></div>
+
 {% endblock pre-content %}
 
-{% block read-buttons %}
-{% endblock read-buttons %}
\ No newline at end of file
+{% block read-buttons %}{% endblock read-buttons %}
+
+{% block extra-script %}
+<script src="http://code.highcharts.com/modules/exporting.js"></script>
+<script>
+$(function () {
+    $('#container').highcharts({
+        title: {
+            text: '{{ baselineterm.name }}'
```

Comment:
If the baseline term name has an apostrophe in it then this could get ugly. You could [escape such values](https://docs.djangoproject.com/en/dev/ref/templates/builtins/#escapejs), but I'm wondering: what do you think about creating a JSON endpoint from which to grab this data?